code/game/objects/items/storage/boxes.dm

```diff
@@ -124,25 +124,41 @@
 // Ordinary survival box
 /obj/item/storage/box/survival/PopulateContents()
 	new /obj/item/clothing/mask/breath(src)
-	new /obj/item/tank/internals/emergency_oxygen(src)
 	new /obj/item/reagent_containers/hypospray/medipen(src)
+	
+	var/mob/living/M = src.loc
+	if(!isplasmaman(M))

```

Comment:
This will runtime if the box is ever instanced not inside a mob, probably resulting in boxes not in mobs missing oxygen tanks.

Reply:
Since when do istype checks runtime if the argument is null and/or the wrong type?

Reply:
I don't think it will runtime, but it definitely doesn't need to be typecasted

Reply:
actual real Maintainer on Discord advised I just use `if(!isplasmaman(loc))`, I tested this and it seemed to work both in my outfit and when spawning survival boxes with the admin debug tools. No runtimes on the debug runtime tool detected.